src/Components/Admin/admin.js

```diff
@@ -582,13 +582,13 @@ export default function Admin() {
                             handleDeleteAssetClick={handleDeleteAssetClick}
                         />
                     </TabPanel>
-                    <TabPanel
+                    {/* <TabPanel
```

Comment:
Just commenting it out in case we want to come back to it. Better than removing imo!